config/setup.go

```diff
@@ -42,7 +42,8 @@ func SetupGRPCConnection(
 	defer timeoutCancel()
 	// Increase the gRPC max message size from the default value of 4MB to 32MB, to match the limit that is set in RDK. This is
 	// necessary for transmitting large pointclouds.
-	maxMsgSizeOption := grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(32 * 1024 * 1024))
+    maxMessageSize := 32 * 1024 * 1024
```

Comment:
Can we please move this into a module top-level constant along with the comment around the message size?

Reply:
Done. Is this what you were looking for?